lib/launch-fastify.js

```diff
@@ -103,13 +104,14 @@ async function launchFastify(file, options, address) {
   const logger = customLogger(mergedOptions, options)
   const fastify = fastifyBuilder({ ...exportFastifyOptions(mergedOptions), logger })
 
+  const openApiSpecification = get(serviceModule.swaggerDefinition, 'openApiSpecification', 'openapi').toLowerCase()
```

Comment:
why mixing lodash with dot notation? For coherence shouldn't it be better doing

```suggestion
  const openApiSpecification = get(serviceModule, 'swaggerDefinition', 'openApiSpecification', 'openapi').toLowerCase()
```

Reply:
I removed lodash and used optional chaining